src/server/services/checkWebsiteStatus.js

```diff
@@ -0,0 +1,52 @@
+const cron = require('node-cron');
+const { getConnection } = require('../db');
+const { logMailer } = require('../logMailer');
+const LogEmail = require('../models/LogEmail');
+const colors = require('colors');
+  
+async function checkWebsite() {
+  const conn = getConnection();
+
+  try {
+    const response = await fetch('https://httpstat.us/500/', { method: 'HEAD' });
+
+    if (response.ok) {
+      console.log('The server is up'.green.inverse);
+    } else {
+      console.error('The server is down'.red.inverse);
+
+      const errorMessage = 'The server at https://httpstat.us/500/ is down.';
+
+      // Log the error in the database
+      const log = new LogEmail(undefined, errorMessage);
+      await log.insert(conn);
+
+      // Send a logging email
+      await logMailer(conn);
+    }
+  } 
+  catch (error) {
+    console.error('Error:', error);
+
+    const errorMessage = `Failed to reach https://httpstat.us/500/. Error: ${error.message}`;
+
+    // Log the error in the database
+    const log = new LogEmail(undefined, errorMessage);
+    await log.insert(conn);
+
+    // Send a logging email
+    await logMailer(conn);
+  } finally {
```

Comment:
My recollection is the conn comes from a pool of connections that can be reused. I cannot see anywhere else that conn.close is called so I don't think this is necessary. Let me know if you think otherwise.

Reply:
You are correct that I 'finally' block isnt necessary, I remove conn.close but forgot to remove this. Thanks for noticing